plutus-ledger-api/src/PlutusLedgerApi/V1/Data/Credential.hs

```diff
@@ -0,0 +1,105 @@
+-- editorconfig-checker-disable-file
```

Comment:
Would you be able to remove this disabler and also format new files with fourmolu?